languages/java/exercises/concept/switch-statement/.docs/after.md

```diff
@@ -0,0 +1,134 @@
+Starting with Java 14 (available as a preview before in Java 12 and 13) it is possible to use the "enhanced" switch implementation.
+
+1. You have the possiblity to assign multiple value in a sigle case.
+   Instead of :
+
+   ```java
+   switch (number) {
+       case 1:
+           //do stuff
+           break;
+       case 2:
+           //do different stuff
+           break;
+       case 3:
+           //do same stuff as case 1
+           break;
+       (...)
+   }
+   ```
+
+   You can do :
+
+   ```java
+   switch (number) {
+       case 1, 3:
+           //do stuff
+           break;
+       case 2:
+           //do other stuff
+           break;
+       (...)
+   }
+   ```
+
+2. The `switch-statement` is now a `switch expression`. What is the difference ?
+   Basicly a statement is expecting some strict logic where an expression can return a value.
+   Instead of :
+
+   ```java
+   String result = "";
+   switch (expression) {
+       case "bob":
+           result = "bob;
+           break;
+       (...)
+   }
+   ```
+
+   You can do :
+
+   ```java
+   String result = switch(expression) {
+       case "bob":
+           yield "bob";
+       (...)
+   }
+   ```
+
+   The [`yield`][yield-keyword] works like a `return` except it's for switch expression. As `yield` terminates the expression `break` is not needed here.
+
+3. An other main difference between the `switch statement` and the `switch expression` : in a `switch expression` you _**MUST**_ cover all the case in a `switch expression`. Either by mutliple `case` or using a `default` statement.
+
+4. You can use `->` instead of `:`. The `->` allow you to not include the `break` keyword. Both notation can be used but in a switch you have to stick with only one.
+
+   ```java
+       switch(expression) {
+           case 1 -> //do something
+           case 2: //do something
+           (...)
+       }
+   ```
+
+   Won't work!
```

Comment:
I'd like to see code that does work too. Maybe you can combine this?

````suggestion
   ```java
       switch(expression) {
           case 1 -> yield "one"           
           case 2 -> yield "two"
           default: yield "other number" // Removing this will result in a compile error
       }
   ```
````